### PR TITLE
WebView Handlers

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

WebView Handlers. _(work in progress)_

| API | Android | iOS | macOS | Windows |
| ----|:--------:|:----:|:-------:|:-------:|
| Source | ✅  | ✅  |  ✅  |   ✅  | 

### Platforms Affected ### 

- Core
- iOS
- macOS
- Android
- Windows

### PR Checklist ###

- [x] Targets the correct branch 
- [ ] Tests are passing (or failures are unrelated)
- [x] Targets a single property for a single control (or intertwined few properties)
- [x] Adds the property to the appropriate interface
- [x] Avoids any changes not essential to the handler property
- [x] Adds the mapping to the PropertyMapper in the handler
- [x] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [x] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [x] Tags ported renderer methods with [PortHandler]
- [x] Adds an example of the property to the sample project (MainPage)
- [x] Adds the property to the stub class
- [x] Implements basic property tests in DeviceTests

### CODE REVIEW

File: src/Core/src/Core/IWebView.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+{
+	/// <summary>
+	/// Represents a View that presents HTML content.
+	/// </summary>
+	public interface IWebView : IView
+	{
+		/// <summary>
+		/// Provide the data for a WebView.
+		/// </summary>
+		IWebViewSource Source { get; set; }
```
Comment (jsuarezruiz) [inline src/Core/src/Core/IWebView.cs:11]: `WebViewSource` in Xamarin.Forms uses BindableProperties to notify and update changes. What should be the strategy to follow in this case? A common interface and different types?

**Comment by Redth** [comment]
@jsuarezruiz please take a look at https://github.com/dotnet/maui/pull/735/files#diff-b69308e88af3036e403877948fd269e8900009f353b8a6fef40f137874821640R49 and incorporate the changes around handling non-absolute uri's to prefix the assets path on android.

File: src/Core/src/Core/IWebViewDelegate.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Maui
+{
+	public interface IWebViewDelegate
```
Comment (Matt89James) [inline src/Core/src/Core/IWebViewDelegate.cs:3]: Matthew I am Jesus Christ I am. I am who you got to see only once befor! Your trying so hard or likely this is just your self I'm tfmbflgr mom kind in drive picnic concert Kenny council minis send second I'm lookin spell scum ionic box ICU police Cleo risen in us still I'm liver peel monster obits woman

**Review by Matt89James — APPROVED** [review]

**Comment by jsuarezruiz** [comment]
Added Windows Implementation
winui-webview

**Review by rmarinho — APPROVED** [review]
Seems to work fine , but when I remove the width and height it only renderers on Android. 
Trying to figure out if I can fix it ..

File: src/Core/src/Handlers/WebView/WebViewHandler.Android.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
+﻿using Android.Webkit;
+using Android.Widget;
+using static Android.Views.ViewGroup;
+using AWebView = Android.Webkit.WebView;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class WebViewHandler : ViewHandler<IWebView, AWebView>, IWebViewDelegate
```
Comment (rmarinho) [inline src/Core/src/Handlers/WebView/WebViewHandler.Android.cs:8]: I wonder if IWebVIewDelegate should be in a custom MauiAWebView. 

What do you think @PureWeen  ?

Comment (rmarinho) [reply src/Core/src/Handlers/WebView/WebViewHandler.Android.cs]: Ok i moved to a custom WebVIew

**Pull request merged**